dascore/io/core.py

```
@@ -1294,11 +1297,50 @@ def get_format(
+    if not any(gapped):
+        return spool
+    if not split:
+        msg = (
+            "Cannot write patches whose dimensional coordinates contain "
+            "gaps (segmented coordinates); a written patch must be "
+            "contiguous. Pass split=True to write each contiguous section "
+            "as its own patch, or split explicitly with patch.split_gaps()."
+        )
+        raise ParameterError(msg)
+    patches = []
+    for patch, has_gaps in zip(spool, gapped):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)